From e7883b7d6428662077fa88c8dfa4e0fdee28ff46 Mon Sep 17 00:00:00 2001
From: "Azamat H. Hackimov" <azamat.hackimov@gmail.com>
Date: Fri, 22 Apr 2016 13:42:26 +0500
Subject: [PATCH] Fixing underlinking for gcc 4.9

Allegro4 backend uses XGrabPointer() from libX11.
Added ${X11_LIBRARIES} to resolve underlinking.
---
 src/she/CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/she/CMakeLists.txt b/src/she/CMakeLists.txt
index cddf5bc..1841aa6 100644
--- a/src/she/CMakeLists.txt.orig	2016-04-30 10:36:25.198000000 +0000
+++ b/src/she/CMakeLists.txt	2016-04-30 10:37:01.163000000 +0000
@@ -241,7 +241,8 @@
   target_link_libraries(she
     ${LOADPNG_LIBRARY}
     ${LIBALLEGRO4_LINK_FLAGS}
-    ${DXGUID_LIBRARIES})
+    ${DXGUID_LIBRARIES},
+    ${X11_LIBRARIES})
 endif()
 
 if(USE_SKIA_BACKEND)
-- 
2.7.3

